langchain4j-mariadb/src/test/java/dev/langchain4j/store/embedding/mariadb/MariaDbEmbeddingIndexedStoreIT.java

```diff
@@ -0,0 +1,49 @@
+package dev.langchain4j.store.embedding.mariadb;
+
+import static org.testcontainers.shaded.org.apache.commons.lang3.RandomUtils.nextInt;
+
+import dev.langchain4j.data.segment.TextSegment;
+import dev.langchain4j.model.embedding.EmbeddingModel;
+import dev.langchain4j.model.embedding.onnx.allminilml6v2q.AllMiniLmL6V2QuantizedEmbeddingModel;
+import dev.langchain4j.store.embedding.EmbeddingStore;
+import dev.langchain4j.store.embedding.EmbeddingStoreWithFilteringIT;
+import org.testcontainers.containers.MariaDBContainer;
+import org.testcontainers.junit.jupiter.Container;
+import org.testcontainers.junit.jupiter.Testcontainers;
+
+@Testcontainers
+class MariaDbEmbeddingIndexedStoreIT extends EmbeddingStoreWithFilteringIT {
+
+    @Container
+    static MariaDBContainer<?> mariadbContainer =
+            new MariaDBContainer<>(MariaDBImage.DEFAULT_IMAGE);
```

Comment:
nit: Wouldn't it be better to create and start MariaDBImage (it can be called MariaDB then) container statically, so it is reused across all integration tests? Looks like a subject of the follow-up PR.